src/map/modifier.h

```diff
@@ -204,6 +204,8 @@ enum MODIFIER
     MOD_CRITHITRATE               = 165, // Raises chance to crit
     MOD_CRIT_DMG_INCREASE         = 421, // Raises the damage of critcal hit by percent %
     MOD_ENEMYCRITRATE             = 166, // Raises chance enemy will crit
+    MOD_MAGIC_CRITHITRATE         = 541, // Raises chance to magic crit
+    MOD_MAGIC_CRIT_DMG_INCREASE   = 542, // Raises damage done when criting with magic
```

Comment:
Same here, 562/563. I'd appreciate it if you didn't make the same mistake I did by incrementing the spare mods to 564/565 as well.